### ISSUE TITLE
[FEATURE] `wash get inventory` output is unreadable

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wasmCloud#2398]
### Affected project(s)

- [ ] documentation
- [ ] examples
- [ ] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Is your feature request related to a problem? Please describe.

Looking at the inventory of hosts is one of the most common operations for `wash`. However, when using file references or long generated component IDs, it becomes pretty unreadable.
![image](https://github.com/wasmCloud/wasmCloud/assets/12040685/be75a3ac-05f7-4b2d-a3b5-7da861ae5c05)
![image](https://github.com/wasmCloud/wasmCloud/assets/12040685/ae0d7f2b-1342-48b7-afe4-9157a4ebcff8)

Taking a look at other infrastructure tools, like kubectl and nomad, none of them quite overflow the terminal like wash. I think this is largely due to the fidelity of information, and with wash displaying the full image reference really just isn't quite necessary.

### Describe the solution you'd like

I would love for the host inventory to display just the right amount of information for the human readable format. Just like kubernetes, we can hide all of the other information behind the `json` flag for when it's necessary.
![image](https://github.com/wasmCloud/wasmCloud/assets/12040685/121bb53b-5c89-471c-b15a-d6aa4ba81fd7)

I propose that the host inventory display the following information:
- Host ID
- Host friendly name
- Host Labels
- Components list
  - Component ID
  - Name
  - Max count
- providers list
  - Provider ID
  - Name

I could even be convinced to drop the friendly name / host labels for the inventory operation, but it can be useful at a glance to verify that, for example, your application is spread properly across availability zones.

### Describe alternatives you've considered

- We could push host information into the `wash get hosts` command to keep the terminal more clean
- We could be smarter with our table library to fit to the width of the terminal and truncate the image reference. This could work, but a truncated image reference or ID is pretty unhelpful

**Comment by ossfellow** [comment]
I can take this; please assign it to me.

**Comment by ossfellow** [comment]
Hello @brooksmtownsend,
Does this look okay? If it does, then, I'll create the PR; otherwise let me know of any other improvements you'd like to see:

![image](https://github.com/wasmCloud/wasmCloud/assets/50174679/78ac1a7a-d801-4254-8b2c-8f89132a3331)

**Comment by brooksmtownsend** [comment]
@ossfellow this looks wonderful! 😲

**Comment by vados-cosmonic** [comment]
Hey @brooksmtownsend this was resolved right?

**Comment by brooksmtownsend** [comment]
Yep! Coming soon in a wash 0.30.0 near you. Thanks again @ossfellow
